gm4_midnight_menaces/data/gm4_illusioner_nights/functions/event.mcfunction

```diff
@@ -2,4 +2,5 @@
 # at world spawn
 # called by midnight menaces base if this expansion was selected. pulsed every 0.8 seconds throughout the night.
 
-execute unless score illusioner_nights gm4_menace_data matches 1 run function gm4_illusioner_nights:initialise
+execute as @e[type=#gm4_illusioner_nights:illusioner_convertable,tag=!gm4_mm_illusioner_checked,x=0,predicate=gm4_illusioner_nights:illusioner_convert_chance,nbt={PersistenceRequired:0b}] unless data entity @s DeathLootTable at @s run function gm4_illusioner_nights:illusioner_transform
+tag @e[type=#gm4_illusioner_nights:illusioner_convertable] add gm4_mm_illusioner_checked
```

Comment:
Since you're changing the marking of entities from a scoreboard to a tag, how does this work for existing marked entities?

Reply:
The score and tag are not related. The score was used to track if the night just started; to initialize the slow clock, while the tag is used for marking. 

The old version of the module didn't convert mobs, it rather spawned them with its own (flawed) spawning system.